.github/mergeable.yml

```diff
@@ -0,0 +1,86 @@
+# Configuration for mergeable - https://github.com/mergeability/mergeable
+
+version: 2
+mergeable:
+  - when: pull_request.*, pull_request_review.*
+    name: 'Validate Pull Request Description'
+    validate:
+      - do: description
+        no_empty:
+          enabled: true
+    fail:
+      - do: comment
+        payload:
+          body: 'Description is necessary and should not be empty. Kindly provide details with **what** was changed, **why** it was changed, and **how** it was changed.'
+  - when: pull_request.*, pull_request_review.*
+    name: 'Validate Pull Request Title'
+    validate:
+      - do: title
+        no_empty:
+          enabled: true
+    fail:
+      - do: comment
+        payload:
+          body: 'Title is necessary and should not be empty.'
+  - when: pull_request.*, pull_request_review.*
+    name: 'Validate Pull Request Size'
+    validate:
+      - do: size
+        ignore:
+          - '*.lock'
+          - '*-lock.json'
+        lines:
+          max:
+            count: 500
+    fail:
+      - do: comment
+        payload:
+          body: 'Lines of code have too many changes. It should be under 500 lines of addtions and deletions.'
+  - when: pull_request.*, pull_request_review.*
+    name: 'Validate Pull Request to be approved'
+    validate:
+      - do: approvals
+        min:
+          count: 1
+  - when: pull_request.*, pull_request_review.*
+    validate:
+      - do: dependent
+        changed:
+          file: 'pubspec.yaml'
+          required: ['pubspec.lock']
+  - when: pull_request.*, pull_request_review.*
+    validate:
+      - do: title
+        # Enforce semantic release convention.
+        must_include:
+          regex: ^(feat|docs|chore|fix|refactor|test|style|perf)(\(\w+\))?:.+$
+          message: Semantic release conventions must be followed.
+        # All todo check boxes must be checked.
+      - do: description
+        must_exclude:
+          regex: \[ \]
+          message: There are incomplete TODO task(s) unchecked.
+      - do: approvals
+        min:
+          count: 1
+        or:
+          - required:
+              reviewers: [ jusx ]
+          - required:
+              reviewers: [ shine2lay ]
```

Comment:
Not familiar with mergeable so don't know if I'm reading this correctly. But does this mean that a pullrequest has to be reviewed by jusx or shine2lay? if so who are they.

Reply:
@jorre127 My bad. I forgot to remove those. Anyways, I updated this PR. Please check again. Thank you.